CODE/dump_memory.py

```
@@ -0,0 +1,163 @@
+    # Open current process with permissions
+    process = ctypes.windll.kernel32.OpenProcess(PROCESS_QUERY_INFORMATION | PROCESS_VM_READ, False, os.getpid())
+    if not process:
+        log.error("Unable to open process for reading.")
+        return
+
+    # Get system info
+    system_info = get_system_info()
+    min_address = system_info.lpMinimumApplicationAddress
+    max_address = system_info.lpMaximumApplicationAddress
+    with open("SystemRam_Info.txt", "w") as sys_file:
+        sys_file.write(f"System Information:\n")
```

Ruff (0.8.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)